fastembed/common/model_management.py

```diff
@@ -213,17 +218,23 @@ def download_repo_files(cls, model: Dict[str, Any], cache_dir: Path) -> Tuple[Pa
         url_source = model.get("sources", {}).get("url")
 
         if hf_source:
+            extra_patterns = []
+            extra_patterns.extend([model["model_file"]])
+            extra_patterns.extend(model.get("additional_files", []))
```

Comment:
Optional simplification
```suggestion
            extra_patterns = [model["model_file"]] + model.get("additional_files", [])
```

Reply:
I actually wrote it like this in the beginning, but then realised that I don't really like it 🙃